pkg/tests/end_to_end_tests/new_migrate_test.go

```diff
@@ -0,0 +1,45 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+package end_to_end_tests
+
+import (
+	"context"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stretchr/testify/require"
+	"github.com/timescale/promscale/pkg/pgmodel"
+	"github.com/timescale/promscale/pkg/pgmodel/common/extension"
+	"github.com/timescale/promscale/pkg/version"
+	"testing"
+)
+
+func TestMigrationFailesIfMultiplePromSchemaMigrationsTables(t *testing.T) {
```

Comment:
Nit: test would be easier to read/understand if you were to run migrate once, add an additional schema table and check that migrate fails with the correct error.

Reply:
I've been fiddling with this for a while, and to be honest there's no easy way to do what you're asking for. As you brought it up as a nit, I'm not sure if it's worth sinking more time into this. I did find a couple of small improvements to this test, which I will push.

Reply:
I did end up figuring out how to do this and implemented it as suggested.